src/Management/ResourceBuilder.php

```diff
@@ -410,6 +412,14 @@ private function buildUser(array $data): User
         ]);
     }
 
+    private function buildOrganization(array $data): Organization
+    {
+        return $this->createObject(Organization::class, [
+            'name' => $data['name'],
+            'sys' => $this->buildSystemProperties($data['sys'])
```

Comment:
That's the reason I usually prefer not to go off the beaten path with custom config and just use a (strict) ruleset, it's easier to avoid that kind of situation 🙂